scripts/exregional_run_analysis.sh

```diff
@@ -304,21 +320,40 @@ fi
 ln_vrfy -snf ${fixgriddir}/fv3_akbk                     fv3_akbk
 ln_vrfy -snf ${fixgriddir}/fv3_grid_spec                fv3_grid_spec
 
-if [ ${BKTYPE} -eq 1 ]; then  # cold start uses background from INPUT
-  ln_vrfy -snf ${bkpath}/gfs_data.tile7.halo0.nc        gfs_data.tile7.halo0.nc_b
-  ln_vrfy -snf ${fixgriddir}/phis.nc                    phis.nc
-  ncks -A -v  phis               phis.nc           gfs_data.tile7.halo0.nc_b
-
-  ln_vrfy -snf ${bkpath}/sfc_data.tile7.halo0.nc        fv3_sfcdata
-  ln_vrfy -snf gfs_data.tile7.halo0.nc_b                fv3_dynvars
-  ln_vrfy -s fv3_dynvars                           fv3_tracer
-
-  fv3lam_bg_type=1
-else                          # cycle uses background from restart
-  ln_vrfy  -snf ${bkpath}/fv_core.res.tile1.nc             fv3_dynvars
-  ln_vrfy  -snf ${bkpath}/fv_tracer.res.tile1.nc           fv3_tracer
-  ln_vrfy  -snf ${bkpath}/sfc_data.nc                      fv3_sfcdata
-  fv3lam_bg_type=0
+if [ ${if_observer} == "True" ]; then
+  echo " Running observer for mem0001"
+
+  restart_prefix=${YYYYMMDD}.${HH}0000.
+  checkfile=${bkpath}/${restart_prefix}fv_core.res.tile1.nc
+  if [ -r "${checkfile}" ]; then
+    print_info_msg "$VERBOSE" "Found ${checkfile}; Use it as background for analysis "
+    ln_vrfy  -snf  ${bkpath}/${restart_prefix}fv_core.res.tile1.nc             fv3_dynvars
+    ln_vrfy  -snf  ${bkpath}/${restart_prefix}fv_tracer.res.tile1.nc           fv3_tracer
+    ln_vrfy  -snf  ${bkpath}/${restart_prefix}sfc_data.nc                      fv3_sfcdata
+    fv3lam_bg_type=0
+  else
+    print_err_msg_exit "$VERBOSE" "Error: cannot find background: ${checkfile}"
+  fi
```

Comment:
Line 324-336.  this logic is already in the prepstart task. We need to follow a similar logic as prod cycles after line 338.

Reply:
ensemble part does not use the "prepstart" task. This task is conducted to produce OMB only, no impact on the rest of the workflow.